cloud/maintenance.md

```diff
@@ -67,4 +67,47 @@ downtime is usually between 30&nbsp;seconds and 5&nbsp;minutes. We endeavor to
 notify you on email ahead of the upgrade if downtime is required, so that you
 can plan accordingly. However, in some cases, we might not be able to do so.
 
+## Upgrade to a new PostgreSQL version
+Timescale currently supports PostgreSQL&nbsp;12 and 13. You can manually upgrade
+to the next PostgreSQL version in the Timescale Cloud dashboard.
+
+Upgrading to a newer version of PostgreSQL allows you to take advantage of new
+features, enhancements, and security fixes. It also ensures that you are using a
+version of PostgreSQL that is compatible with the newest version of TimescaleDB,
+allowing you to take advantage of everything Timescale has to offer. For more
+information about feature changes between versions, see the
+[PostgreSQL release notes][postgres-relnotes] and
+[TimescaleDB release notes][timescale-relnotes].
+
+<highlight type="warning">
+Your Timescale Cloud service is unavailable for use until the upgrade is
+complete. Upgrading can take up to several hours to complete, so we recommend
+that you plan ahead, and upgrade during a time with low usage.
+</highlight>
+
+If you want to try out the upgrade before you run it on your production system,
+you can create a fork of your service, and attempt the upgrade on the fork
+before you start. This can give you a good idea of what happens during an
+upgrade, and how long it might take in your use case. For more infomration about
+creating a fork of your service, see the [Operations
+section][operations-forking].
+
+<procedure>
+
+### Upgrading to a new PostgreSQL version
+1.  In the Timescale Cloud console, navigate to `Services` and click the service
+    you want to adjust.
+1.  Navigate to the `Operations` tab, and go to the `Maintenance` section.
+1.  If a new PostgreSQL version is available, click the `Upgrade` button, and
+    confirm that you are ready to start the upgrade. Your Timescale Cloud
```

Comment:
Should  the following sentence go as a note at the beginning of the section: " Your timescale Cloud service is unavailable for use until the upgrade is complete."

Reply:
It's in an admon in the introductory part. Do you think it needs to go within the procedure block as well?

Reply:
> It's in an admon in the introductory part. Do you think it needs to go within the procedure block as well?

May be not !!! Users tend to glance through the introduction paras and jump straight to the steps unless, some admon are called out.

Reply:
True, but I also don't like peppering admons through a procedure. There's also the whole problem of people glazing straight over admons as well. I think having it as prose in the step is a way to catch anyone who missed the warning above.